ds701_book/21-Networks-I.qmd

```diff
@@ -752,64 +874,78 @@ nx.draw_networkx(F_random, ax = ax2,
                  pos = nx.spectral_layout(F_random),
                  with_labels=False, alpha=.8, linewidths=2)
 plt.axis('off')
-plt.title('Same Density Random -- Spectral Layout', size = 16);
+plt.title('Same Density Random -- Spectral Layout', size = 16)
+plt.show()
 ```
 
-With this layout, we can start to understand the structure of the network --
+---
+
+With the spectral layout, we can start to understand the structure of the network.
 
-see clusters of teams that correspond to conferences, --
+We see clusters of teams that correspond to conferences, and understand that there are not too many high-degree nodes.
+
+
+```{python}
+#| fig-align: center
+plt.figure(figsize = (5, 7))
+nx.draw_networkx(football,
+                 node_size=35, 
+                 edge_color='gray', 
+                 pos = nx.spectral_layout(football),
+                 with_labels=False, alpha=.8, linewidths=2)
+plt.axis('off')
+plt.title('Title 1 Football -- Spectral Layout', size = 16)
+plt.show()
+```
 
-and understand that there are not too many high-degree nodes.
 
-### Characterizing Graphs
+## Metrics
 
-Another way to undersand network structured data is to look at important metrics.
+Another way to understand network structured data is to look at important metrics.
 
-For example, we can start with __clustering coefficient:__
+For example, we can start with the __clustering coefficient:
```

Comment:
Missing the other __ after coefficient to make bold

Reply:
Fixed.